server/viame_server/serializers/viame.py

```diff
@@ -3,11 +3,49 @@
 """
 import csv
 import re
+from dataclasses import dataclass, asdict, field
+from typing import List, Dict, Tuple, Optional, Union, Any
 
 from girder.models.file import File
 
 
-def _deduceType(value):
+@dataclass
+class Feature:
+    """Feature represents a single detection in a track."""
+
+    frame: int = None
+    bounds: List[float] = None
+    head: Optional[Tuple[str, str]] = None
+    tail: Optional[Tuple[str, str]] = None
```

Comment:
should be `[number, number]` -- it's an (x, y) point

Reply:
Good catch, thanks.

Reply:
Is `int` sufficient? Or would it require `float`?